### PR TITLE
test(discover): Fix Percy snapshots

### PR DESCRIPTION
**Pull request by lynnagara**
Make Percy render data from Snuba

### CODE REVIEW

File: tests/acceptance/test_organization_discover.py [hunk 1]
```diff
@@ -1,33 +1,57 @@
 from __future__ import absolute_import
 
+from datetime import datetime, timedelta
+
 from sentry.testutils import AcceptanceTestCase
 
 
 class OrganizationDiscoverTest(AcceptanceTestCase):
```
Comment (markstory) [inline tests/acceptance/test_organization_discover.py]: If you mixin SnubaTestCase the `store_event` method will go into snuba and the test will auto-skip should snuba not be running.

**Review by markstory — APPROVED** [review]

**Comment by lynnagara** [comment]
Damn, query results still don't appear in Percy

**Comment by lynnagara** [comment]
The notice that no events were sent is gone though, which is more correct

**Comment by markstory** [comment]
@lynnagara You might be running into datetime problems. Our acceptance test case mock's django's time to be in 2013. However, when snuba ingests data it does not have the same mocked time. In tests I worked on I had to do the following

```python
    @patch('django.utils.timezone.now')
    def test_with_results(self, mock_now):
        mock_now.return_value = datetime.utcnow().replace(tzinfo=pytz.utc)
```

And then insert data to snuba. Remocking timezone.now allowed snuba and sentry to agree on what time it was.

**Comment by lynnagara** [comment]
@markstory Nice, giving that a shot 🙏

**Comment by lynnagara** [comment]
That was it! @markstory - thanks!

**Pull request merged**